### PR TITLE
Notify group curators of memberships that require renewal.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: mozillians/jinja2/groups/email/notify_curator_renewal.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{% trans %}
+The membership of {{ member_full_name}} to group
+"{{ group_name }}" is about to expire in two weeks.
+Please take the appropriate action to ensure that user
+remains subscribed to this group.
+{% endtrans %}
+
+{% trans %}
+You can view the group membership here:
```
Comment (akatsoulas) [inline mozillians/jinja2/groups/email/notify_curator_renewal.txt]: Maybe changing this to `group membership status`?

File: mozillians/jinja2/groups/email/notify_curator_renewal.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% trans %}
+Hello {{ curator_full_name }},
+{% endtrans %}
+
+{% trans %}
+The membership of {{ member_full_name}} to group
+"{{ group_name }}" is about to expire in two weeks.
+Please take the appropriate action to ensure that user
```
Comment (akatsoulas) [inline mozillians/jinja2/groups/email/notify_curator_renewal.txt]: Probably it makes sens to add here the name of the user and a link to the profile

File: mozillians/groups/tasks.py [hunk 3]
```diff
@@ -182,9 +184,32 @@ def notify_membership_renewal():
 
             subject_msg = unicode('[Mozillians] Your membership to group "{0}" is about to expire')
             subject = _(subject_msg.format(membership.group.name))
-            message = template.render(ctx)
+            message = member_template.render(ctx)
             send_mail(subject, message, settings.FROM_NOREPLY, [membership.userprofile.email])
```
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: Shall we assign the `send_mail` to celery?

File: mozillians/groups/tasks.py [hunk 4]
```diff
@@ -182,9 +184,32 @@ def notify_membership_renewal():
...
 
+            invites_query = {
+                'group': membership.group,
+                'redeemer': membership.userprofile,
+                'accepted': True
+            }
+
+            invites = Invite.objects.filter(**invites_query)
```
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: Why are we focusing on invites? What about the memberships of a reviewed group?

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/groups/tasks.py (hunk 3 above)
Comment (johngian) [inline mozillians/groups/tasks.py]: Its already inside a task so its non blocking. Also we are already doing the same above.

File: mozillians/jinja2/groups/email/notify_curator_renewal.txt (hunk 1 above)
Comment (johngian) [inline mozillians/jinja2/groups/email/notify_curator_renewal.txt]: I think ``status`` is something that we use internally and might more confusing for users.

File: mozillians/groups/tasks.py (hunk 3 above)
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: I meant to use celery for both tasks. The task itself is no blocking but maybe it makes sense to spawn different tasks for the emails. Anyways, this is really nit

**Pull request merged**